src/2/world6.md

```diff
@@ -1,263 +1,263 @@
-One could only wonder just when it has manifested.
-In fallen angel's hand was tightly grasped dark red coloured sword, it's colour made it look like it was soaked in blood.
+It was a flamberge, whose hilt looked like the imitation of a wing.
+One could only wonder just when it had manifested.
+In the fallen angel's hand was a tightly grasped dark red coloured sword. Its colour made it look like it was soaked in blood.
 
-Lighting attack was only a distraction.
+The lightning attack was just a distraction.
 By distracting the elven shrine maiden with enormous light, he was able to quickly approach her.
-And then cut her magic tool with the sword.
+And then he cut her magic tool with the sword.
 
-"...I-impossible. For magic sword... to pierce through my spirit garments...!"
+"...I-impossible. For a magic sword... to pierce through my spirit garments...!"
 
-Elf's shoulder was cut.
-Fallen angel's sword was already able to cut elven girl skin, covered by protective garments.
-Before their eyes was terrible scene of emerald blood flowing from girl's shoulder.
+The elf's shoulder was cut.
+The fallen angel's sword was already able to cut the elven girl's skin, covered by protective garments.
+Before their eyes was the terrible scene of emerald blood flowing from the girl's shoulder.
 
 "Reiren!?"
 
-Jeanne reflexively caught in her arms falling down elven girl.
+Jeanne reflexively caught the falling down elven girl in her arms.
 
 _TL TODO:_ I'm a bit unsure what author means with `それが仇`, `仇` reads as _ada_ and can mean various things like: ill intent, grudge or harm.
 
 [So foolish.]
 
 Jeanne screamed.
-While holding the elf, she was attacked by fallen angel's sword.
+While holding the elf, she was attacked by the fallen angel's sword.
 The knight's armour was cut like a thin paper, from her back to the side.
 
 [Do you wish to be cut down while protecting this elf?]
 
 "...!"
 
-Both elf and human fell to their knees.
-Both girls, unable to muster a single word, were facing down the floor.
+Both the elf and the human fell to their knees.
+Both the girls, unable to muster a single word, were facing down towards the floor.
 
 [Disappear.]
 
 "Alfreyja!"
 
-Towards the angel with dark bloody flamberge, that wounded both girls, Kai sent vertical slash with his own sword.
+Towards the angel with his dark bloody flamberge that wounded both the girls, Kai sent a vertical slash with his own sword.
 
 [Hou? Your sword...]
 
-While deflecting the sword, fallen angel laughed.
+While deflecting the sword, the fallen angel laughed.
 
 [I see it, now.]
 
-"I get it, you flamberge...!"
+"I get it, your flamberge...!"
 
-Both human and fallen angel understood at the same time.
-The moment their swords crossed, they both caught the glimpse of each other swords power.
+Both the human and the fallen angel understood at the same time.
+The moment their swords crossed, they both caught the glimpse of each other's sword's power.
```

Comment:
It might sound bad, but I think grammatically it's correct. It's like the "sword" belongs to each other, and the "power" belongs to the sword. I'm having a hard time rephrasing it...
But, indeed, "the glimpse" should probably be "a glimpse"!
> The moment their swords crossed, they both caught a glimpse of each other's sword's power.

Reply:
Ok